frontend/src/components/Sidebar/Sidebar.jsx

```
@@ -1,18 +1,420 @@
+            <button
+              className="absolute top-4 right-4 text-gray-400 hover:text-white text-xl"
+              onClick={() => setShowInvite(false)}
+            >
+              ×
+            </button>
+            <h2 className="text-white text-2xl font-bold mb-2">
+              Invite people
+            </h2>
+            <div className="mb-4">
+              <label className="block text-white mb-1">Invite by email</label>
+              <input
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)